reporter/src/main/kotlin/Reporter.kt

```diff
@@ -20,11 +20,22 @@
 package org.ossreviewtoolkit.reporter
 
 import org.ossreviewtoolkit.model.AnalyzerResult
+import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.OrtResult
 import org.ossreviewtoolkit.model.ScanRecord
+import org.ossreviewtoolkit.model.config.ScopeExcludes
 
 import java.io.OutputStream
 import java.util.ServiceLoader
+import java.util.SortedMap
+
+internal val SCOPE_EXCLUDE_LIST_COMPARATOR = compareBy<Map.Entry<String, ScopeExcludes>>(
+    { it.value.isNotEmpty() }, { it.key }
+)
+
+internal val SCOPE_EXCLUDE_MAP_COMPARATOR = compareBy<Map.Entry<Identifier, SortedMap<String, ScopeExcludes>>>(
+    { it.value.values.isNotEmpty() }, { it.key }
+)
```

Comment:
I'd prefer to put this in a `Comparators.kt` file in the utils package.

Reply:
Done.